tests/foreman/ui/test_contentview.py

```diff
@@ -4123,3 +4123,44 @@ def test_positive_depsolve_with_module_errata(session, module_org):
             assert items['Name'] in rpm_pack
         assert result['module_streams']['table'][0]['Name'] == mod_stream
         assert result['errata']['table'][0]['Errata ID'] == settings.repos.yum_10.errata[0]
+
+
+@pytest.mark.tier2
+def test_positive_filter_by_pkg_group_name(session, module_org):
+    """Publish a filtered version of a Content View filtered using package group name.
+
+    :id: c7021f46-0168-44f7-a863-4aa34533efdb
+
+    :expectedresults: Published Content View's contents is filtered
+                based on package group.
+
+    :CaseImportance: Medium
+    """
+
+    filter_name = gen_string('alpha')
+    repo_name = gen_string('alpha')
+    package_group = 'birds'
+    expected_packages = (('cockateel'), ('duck'), ('penguin'), ('stork'))
+    create_sync_custom_repo(module_org.id, repo_name=repo_name)
+    repo = entities.Repository(name=repo_name).search(query={'organization_id': module_org.id})[0]
```

Comment:
Consider using `default_sat.api` to access these nailgun entity classes.

Reply:
OK, doing it now